files/routes/front.py

```diff
@@ -384,27 +388,31 @@ def random_user(v):
 @app.get("/comments")
 @auth_required
 def all_comments(v):
-	try: page = max(int(request.values.get("page", 1)), 1)
-	except: page = 1
-
-	sort=request.values.get("sort", "new")
-	t=request.values.get("t", defaulttimefilter)
-
-	try: gt=int(request.values.get("after", 0))
-	except: gt=0
-
-	try: lt=int(request.values.get("before", 0))
-	except: lt=0
-
-	idlist = get_comments_idlist(v=v, page=page, sort=sort, t=t, gt=gt, lt=lt)
-	comments = get_comments(idlist, v=v)
-
+	page = request.values.get("page", 1, int)
```

Comment:
this should still have a max in here

Reply:
Done, the Postgres driver complained when it got a negative offset and crashed the request. Good find